packages/lesswrong/server/fmCrosspost/types.ts

```diff
@@ -39,38 +28,31 @@ export const validateUnlinkCrossposterPayload = (payload: unknown): payload is U
   return true;
 }
 
-export type UpdateCrosspostPayload = {
-  postId: string,
-  draft: boolean,
-  deletedDraft: boolean,
-  title: string,
-}
+export type UpdateCrosspostPayload = DenormalizedCrosspostData & { postId: string }
 
 export const validateUpdateCrosspostPayload = (payload: unknown): payload is UpdateCrosspostPayload => {
-  if (
-    !hasStringParam(payload, "postId") ||
-    !hasBooleanParam(payload, "draft") ||
-    !hasBooleanParam(payload, "deletedDraft") ||
-    !hasStringParam(payload, "title")
-  ) {
-    throw new MissingParametersError(["postId", "draft", "draftDeleted", "title"], payload);
+  if (!hasStringParam(payload, "postId") || !isValidDenormalizedData(payload)) {
+    throw new MissingParametersError(["postId", ...denormalizedFieldKeys], payload);
```

Comment:
[Comment to self] This change was motivated by the fact that previously you were copy-pasting across 4 lines, and now you have too many denormalized fields to do that, which is what motivated a bunch of this PR.

Reply:
Yep, exactly. Making these changes also actually revealed an active bug where `deletedDraft` is missing from on of the places we need to check the denormalized data, which should be fixed now.